README.md

```diff
@@ -91,7 +91,20 @@ make cluster/prepare/local
 ```
 
 ### Configuring Github OAuth
-Log in to RHSSO (see above) and click `Identity Providers` in the left sidebar. In the Github identity provider, find the Redirect URI and paste this URL into the Homepage URL and Authorization callback URL fields of your Github OAuth app.
+
+Follow [docs]((https://docs.openshift.com/container-platform/4.1/authentication/identity_providers/configuring-github-identity-provider.html#identity-provider-registering-github_configuring-github-identity-provider)) on how to register a new Github Oauth application and add the necessary authorization callback URL for your cluster as outlined.
```

Comment:
```suggestion
Follow [docs](https://docs.openshift.com/container-platform/4.1/authentication/identity_providers/configuring-github-identity-provider.html#identity-provider-registering-github_configuring-github-identity-provider) on how to register a new Github Oauth application and add the necessary authorization callback URL for your cluster as outlined.
```

Reply:
@grdryn You'd think that I'd at least sanity check these changes :)

Fixed